how_to_guides/set_up_status_cake.md

```diff
@@ -0,0 +1,79 @@
+# How to set up status cake with your service
+
+This guide outlines the basic requirements to set up status cake to monitor your website up time. Status cake works by making requests to your endpoint at given intervals and then sending a notification if it fails to receive a success HTTP status code. It can send notifications to email addresses and slack channels etc.
+
+## Request account access
+
+The first step is to request access using your .education.gov.uk email address on service now. You can use [this link](https://dfe.service-now.com.mcas.ms/serviceportal?id=sc_cat_item&sys_id=e7a004df1b399c502fe864606e4bcb21) to request access. Make 2 separate requests, 1 for `Request Account Access` and another for `Request API key`. The API key may not be needed to set up a basic uptime test but may be required for more advanced scenarios.
+
+## Set up a health check endpoint in your project
```

Comment:
I'm not 100% sure that this is the best thing to check. StatusCake should give an accurate picture of if the website is up from a user's point of view. With this health check it may be that the health check passes, but that rendering the front page of the service fails as it has more dependencies and code executed. We should research this further and discuss this on Slack.

Reply:
Agreed. If you could post in the Slack channel we can agree an approach there.